packages/block-library/src/columns/style.scss

```diff
@@ -10,6 +10,8 @@
 }
 
 .wp-block-column {
+	margin-bottom: 1em;
+
```

Comment:
If you do get a minute @getdave maybe just get rid of this blank line between properties?

Reply:
Done